app/views/member_profiles/show.html.erb

```diff
@@ -29,7 +29,7 @@
 
     <button class="btn">Edit Account Settings</button>
     <br> <br>
-    <button class="btn">Edit Password</button>
+    <%= button_to "Edit Password", edit_member_profiles_password_url, method: :get, class: 'btn' %>

```

Comment:
Since this is a GET request, I think that `link_to` can do the job here:

```suggestion
    <%= link_to "Edit Password", edit_member_profiles_password_url, class: 'btn' %>
```

The CSS should be able to make a link look like a button.